.changeset/forty-onions-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Run the `deleteWithEagerPruningOfChildren` test without transactions, to try to get rid of CI flakiness
```

Comment:
Perhaps an unnecessary changeset, could arguably even be skipped?

Reply:
Removed!